redis.conf

```diff
@@ -1449,15 +1450,16 @@ auto-aof-rewrite-min-size 64mb
 # will be found.
 aof-load-truncated yes
 
-# When rewriting the AOF file, Redis is able to use an RDB preamble in the
-# AOF file for faster rewrites and recoveries. When this option is turned
-# on the rewritten AOF file is composed of two different stanzas:
-#
-#   [RDB file][AOF tail]
-#
-# When loading, Redis recognizes that the AOF file starts with the "REDIS"
-# string and loads the prefixed RDB file, then continues loading the AOF
-# tail.
+# The BASE AOF encode type.
```

Comment:
```suggestion
# The BASE AOF encoding type.
```

Reply:
@yoav-steinberg i'm not sure which version did you review?
IIUC we eliminated the references to uppercase BASE and INCR in the docs.
maybe your phrasing comments are still applicable, but maybe you should dismiss them and make new suggestions base on the latest?